Automattic/wp-calypso
# Reader post cards - remove visit and views links from reader post actions

Related to # pe7F0s-1fb-p2

## Proposed Changes

* Removes visit and views links from reader post cards.

Before:
<img width="481" alt="Screenshot 2023-09-15 at 9 21 18 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/b64477aa-74bd-4bf1-b890-94b1187f4104">


After:
<img width="484" alt="Screenshot 2023-09-15 at 9 21 24 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/4ef27137-539e-4313-8c47-246462d36009">



## Testing Instructions



* Run this PR
* Verify post cards in the reader no longer show visit or views buttons.

## Pre-merge Checklist
